utils/PortManagerServer.ts

```diff
@@ -193,13 +193,26 @@ class _PortManagerServer {
     }
   };
 
+  async close(): Promise<void> {
```

Comment:
Adding this helper made the `PortManagerServer` a lot easier to test (failing tests here were why I started this process in the first place)